tests/std/tests/P0896R4_ranges_alg_unique/test.cpp

```diff
@@ -45,6 +45,15 @@ struct instantiator {
             assert(comparisonCounter == size(input) - 1);
         }
 
+        { // Validate already unique output returns empty subrange

```

Comment:
Nitpick: It's slightly confusing to talk about the output being already unique. Saying "input" would make more sense, but the `unique_copy` test comments below say "range" which seems even better here (as `unique` is in-place):
```suggestion
        { // Validate already unique range returns empty subrange
```